plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -85,20 +308,55 @@
         template: "#remote-config-parameters",
         mixins: [countlyVue.mixins.hasDrawers("parameters")],
         components: {
-            drawer: ParametersDrawer
+            drawer: ParametersDrawer,
+            "json-editor": JsonEditor
+
         },
         computed: {
             tableRows: function() {
                 return this.$store.getters["countlyRemoteConfig/parameters/all"];
             }
         },
         methods: {
+            getNumberOfConditionsText: function(conditions) {
+                if (conditions.length === 0) {
+                    return "1 condition";
```

Comment:
if conditions length is 0, shouldn't we return 0 condition?

Reply:
This is for default value too

Reply:
but if conditions are 0, why should the default value be 1 condition? @itsiprikshit

Reply:
default value is being stored seperately in db

Reply:
afaik there is no default value for conditions.
if it was addd later, please point me to the use case and db schema for that. @abhisheksingla02

Reply:
@itsiprikshit  this is not default value for condition but default value field for parameter itself. aacording to design we needed to show both. https://www.figma.com/file/PUjfGUiaVQpSBeS5EWMKYu/countly-components?node-id=11374%3A77951